### DISCUSSION TITLE
Move files as they're downloaded

### DISCUSSION THREAD

**Discussion by costaht** [jmbannon/ytdl-sub#830] [Ideas]
I'd like to suggest a change or feature where files can be moved from their temporary folder to their final folder right after each download is finished.
Let's say you have 5 videos of a single channel to be downloaded, ytdl-sub will first download those files, keep them in their temporary folder and only then move those files to their final destination. This is fine if your temporary destination is a HD, but if you're using tmpfs you'll probably start getting some full disk errors later or sonner.
Using tmpfs is a common practice between those trying increase performance and prolonging the life of their SSDs.

Thanks
(tijuco from discord)

**Comment by jmbannon** [comment]
Videos are moved after all edits are done, they do not accumulate in the working dir
